web/src/routes/(app)/account/jobs/+page.svelte

```diff
@@ -5,6 +5,8 @@
 
 	export let data: PageData;
 
+	$: ({ supabase, profile } = data);
```

Comment:
Let's also pull `jobs` out of `data`
```suggestion
	$: ({ supabase, profile, jobs } = data);
```